src/promptflow-devkit/promptflow/_sdk/_pf_client.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, **kwargs):
         # when this is set, telemetry from this client will use this user agent and ignore the one from OperationContext
         self._user_agent_override = kwargs.pop(USER_AGENT_OVERRIDE_KEY, None)
         self._connection_provider = kwargs.pop("connection_provider", None)
-        self._config = Configuration(overrides=kwargs.get("config", None) or {})
+        self._config = Configuration(overrides=kwargs.get("config", None) or {},
```

Comment:
Are we going to do our own validation in the evaluate API?

Reply:
Added